intertidal/elevation.py

```diff
@@ -101,6 +101,9 @@ def ds_to_flat(
     corr : xr.DataArray
         Correlation of NDWI pixel wetness with tide height.
     """
+    # Calculate clear count
+    clear = satellite_ds[index].notnull().sum(dim="time").rename("qa_count_clear")
```

Comment:
Calculate clear count based on the number of non-null values for each pixel through time